src/test/java/studyconnect/member/dto/LoginRequestTest.java

```diff
@@ -0,0 +1,18 @@
+package studyconnect.member.dto;
+
+
+import com.studyconnect.member.dto.LoginRequest;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class LoginRequestTest{
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>WhitespaceAround: '{' is not preceded with whitespace.